resources/deployments/model/deployments_external_test.go

```diff
@@ -859,6 +859,11 @@ func TestDeploymentModelSaveDeviceDeploymentLog(t *testing.T) {
 			testCase.InputDeploymentID,
 			testCase.InputDeviceID).
 			Return(testCase.InputHasDeployment, testCase.InputHasModelError)
+		deviceDeploymentStorage.On("UpdateDeviceDeploymentLogAvailability",
+			testCase.InputDeviceID,
+			testCase.InputDeploymentID,
+			true).
+			Return(nil)
```

Comment:
You're checking success path only right now. How about a scenario in which `UpdateDeviceDeploymentLogAvailability` fails?